### DISCUSSION TITLE
Scrape game info & reviews

### DISCUSSION THREAD

**Discussion by phuctm97** [OnionUI/Onion#1226] [Q&A]
It would be really cool to be able to scrape & display game info & reviews (like how JELOS does it). We already have Scraper feature implemented, it would be more like UI work?

**Comment by schmurtzm** [comment]
Yes, Onion uses MainUI from Miyoo so our possibilities of modifications are very small. 
On the scraper part it's possible (may be I will add the scraping of manuals as it is supported by Onion). 
But to have more like game infos and reviews it would require that Onion has its own UI 🤞🏻.
